lib/node_modules/@stdlib/math/stats/kde2d/lib/pick-bandwidth.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+// MODULES //
+var stdev = require( './stdev.js' );
+var range = require( './range.js' );
+var iqr = require( './iqr.js' );
+
+// MAIN //
+/**
+* Computes the rule-of-thumb bandwidth for given data
+*
+* @private
+* @param {NumericArray} arr - input array
+* @returns {number} bandwidth choice
+*
+* @example
+* var arr = [ 2, 3, 4, 3, 5 ];
+* var out = pickBandwidth( arr );
+* // returns ~0.094809
+*/
+
+function pickBandwidth(data) {
+	var frontConstant;
+	var minElement;
+	var sigmaHat;
+	var bHat;
+	var iqrVal;
+	var n;
+
+	// first get the constant terms
+	sigmaHat = stdev(data);
+	iqrVal = iqr(data);
+	n = data.length;
+
+	// frontConstant is all terms not in the min() call
+	frontConstant = 1.06 * Math.pow(n, -1/5);
```

Comment:
We don't use built-in math library functions. These should be replaced with `@stdlib` implementations.

Reply:
To provide a bit of background on this: Athan gave a talk at Node Interactive last year explaining the rationale of not using any of the native `Math` functions and instead rolling our own: https://www.youtube.com/watch?v=03WhsgTpp7g